indico/modules/rb_new/client/js/util.jsx

```diff
@@ -133,3 +137,51 @@ export function boolStateField(name) {
 export function isDateWithinRange(date, dateRange, _toMoment) {
     return date && dateRange.filter((dt) => _toMoment(dt).isSame(date, 'day')).length !== 0;
 }
+
+
+export function PopupParam({content, children}) {
```

Comment:
nothing for now but I have the feeling we soon need to split this file. We now have various utils and a component in here...